### DISCUSSION TITLE
Proyecto

### DISCUSSION THREAD

**Discussion by JulianaCadavid** [rramosp/ai4eng.v1#86] [00 - General]
Buenas tardes profe, lo que pasa es que no quedé cómoda con ninguna base de datos ya que el monitor me dio otras recomendaciones, me podría recomendar algunas de estas:
https://www.kaggle.com/c/mercari-price-suggestion-challenge/overview/evaluation
https://www.kaggle.com/c/jigsaw-toxic-comment-classification-challenge/code
https://www.kaggle.com/c/talkingdata-adtracking-fraud-detection/data?select=train_sample.csv
https://www.kaggle.com/c/santander-value-prediction-challenge/data
https://www.kaggle.com/c/santander-customer-transaction-prediction/data
Espero que alguna de estas si sea la ideal para el curso. Saludos.

**Comment by Felipe-RA** [comment]
Saludos Juliana.

Pasate por la clase mañana y los miramos
